src/libraries/System.Text.RegularExpressions/gen/System.Text.RegularExpressions.Generator.csproj

```diff
@@ -11,7 +11,7 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <UsingToolXliff>true</UsingToolXliff>
     <CLSCompliant>false</CLSCompliant>
-    <NoWarn>$(NoWarn);CS0436;CS0649</NoWarn>
+    <NoWarn>$(NoWarn);CS0436;CS0649;CS0809</NoWarn>
```

Comment:
Where was the CS0809 error?

Reply:
@stephentoub , yes

Reply:
If that were the case I'd expect CI to be failing everywhere, and it's not.  Please revert this regex change.

Reply:
I removed commit with "NoWarn" changes by force-push, but as I already wrote, I have build error on VS (see screenshots), but [build.cmd](https://github.com/Maximys/runtime/blob/bugfix/30024-problem-with-concurrency-typeDescriptor-getconverter/build.cmd) finished sucessfully after removing _"artifacts"_ folder![Screen](https://user-images.githubusercontent.com/6498433/236393830-7e6fbd80-128f-4bab-b9e9-e7f697538633.PNG)

Reply:
It's possible there's some issue with compilation of the project in the IDE, though I haven't seen that.  But "[build.cmd](https://github.com/Maximys/runtime/blob/bugfix/30024-problem-with-concurrency-typeDescriptor-getconverter/build.cmd) finished sucessfully after removing "artifacts" folder" is what I was referring to when I suggested that doing a clean build should succeed.  Thank you for reverting the change.